### ISSUE TITLE
Tuberculosis Classification DL | 1. Dataset Prep

### ISSUE THREAD

**Issue by ankit1576** [SrijanShovit/HealthLearning#62]
### Is your feature request related to a problem? Please describe.

Tuberculosis is a conveyance illeness that occurs ailing health and death of millions So i would like to Create a classification model using Deep learning techniques

### Describe the solution you'd like

X-ray examination is cosidered to be the most commonly used because of its low cost , wide range of application and fast speed so going to  detect the features from X-ray is ours top priority

### Describe alternatives you've considered

 Going to proposed a model with deep learning algorithm  CNN with some basics layers like conv2D,maxpooling2d,flattendense or we can use Yolo as well or use which works best For this tuberculosis classification we are going to use chest X-Ray image dataset  and aim to achieve high accuracy

### Additional context

Steps to be followed:-

1-Dataset preprocessing and Some preps( includes gathering , cleaning ,understanding , choosing best dataset ,etc)
2-feature engineering and Image preprocessing
3-Augumention (flipping ,cropping ,resizing, rotating ,etc)
4-train and test split 
5-train best model using Training set
6-Validate the model using test set
7-Accuracy ,mAps,Confusion matrix
8- Now inferencing , testing in actual environment

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by SrijanShovit** [comment]
Which dataset you wanna use?

**Comment by ankit1576** [comment]
TBX11K one of the largest TB data  :- https://datasetninja.com/tbx-11k#download

**Comment by SrijanShovit** [comment]
Proceed with 1st 3 steps

**Comment by ankit1576** [comment]
Thank you @SrijanShovit i am working on it

**Comment by VoonnaHarsha** [comment]
@ankit1576  Sir , looking forward to contribute for your task

**Comment by SrijanShovit** [comment]
@ankit1576 any progress?

**Comment by ankit1576** [comment]
sorry for delay @SrijanShovit  i am working on it  step 1- 3  has been done working on further steps

**Comment by SrijanShovit** [comment]
then you can make PR for that part.

**Comment by SrijanShovit** [comment]
@VoonnaHarsha go ahead
